### ISSUE TITLE
Offline Mode - Accessibility

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#11791]
## Objective 

When an accessibility user is in Offline mode, the screen reader should read out the appropriate context of being in offline mode. This includes the internet status banner content, alert messages, and regaining internet access.

## Reference Material:
  

- [Figma documentation](https://www.figma.com/design/TbM6Sf0jOf79f8ffeLCFzn/Offline-mode-%7C-VA-Mobile?node-id=4558-4915&t=CEU2M1OUHs9zxmCY-0)

## Known test data:
  

- Staging user.

## Epic
  
#10884 

  

## Steps
1. Log into the app as a staging user. Ensure the mobile device screen reader is turned ON.
2. Validate the screen reader behaves appropriately in the following scenarios:

| Event | Expected behavior |
| -- | -- |
| Phone loses internet connection | Screen reader announces loss of internet connection |
| User focuses on 'No internet connection' banner | Screen reader reads text content of banner |
| User interacts with the 'No internet connection' banner | The banner expands, displaying the 'Last connected' timestamp. The screen reader reads the expanded banner status and the content of the timestamp.
| User focuses on the 'No content available' alert on a page that does not have any data loaded | The screen reader should read the contents of the alert in focus.
| Phone regains internet connection | Screen reader announces 'Internet connection available'. Page loads intended content and user can resume focus/navigation on page content. |
| Phone regains internet connection while screen reader is focuses on internet status banner | When the user focuses _away_ from the banner after regaining internet access, screen reader should announce 'internet connection available'. Internet status banner is then dismissed and no longer focusable. |
| User interacts with a button or link that displays an 'offline' warning message | The screen reader should read content of the alert/popup message that appears on screen. |

---

## Expected Results
The screen reader should read out the appropriate context as described in the chart above.
---

## Status
- [ ] Pass  
- [ ] Fail  
- [ ] Blocked  

---

## Attachments
---
